package.json

```diff
@@ -13,9 +13,9 @@
     "coffee-loader": "^0.7.3",
     "coffee-script": "1.6.1",
     "css-loader": "^0.28.5",
-    "@edx/edx-bootstrap": "0.3.4",
-    "@edx/paragon": "0.1.0",
-    "@edx/studio-frontend": "0.2.0",
+    "@edx/edx-bootstrap": "^0.4.0",
+    "@edx/paragon": "^0.2.0",
+    "@edx/studio-frontend": "^0.3.0",
```

Comment:
These 2 dependencies are being upgraded elsewhere by @andy-armstrong, I just had to update them here to get my devstack install functioning correctly. Expecting these changes to vanish once I rebase on master (post https://github.com/edx/edx-platform/pull/16302).